src/main/java/org/sonatype/repository/helm/internal/hosted/HostedHandlers.java

```diff
@@ -24,7 +24,7 @@
 import org.sonatype.repository.helm.internal.AssetKind;
 import org.sonatype.repository.helm.internal.util.HelmPathUtils;
 
-import static jline.internal.Preconditions.checkNotNull;
+import java.util.Objects;
```

Comment:
not a big deal, but in other places in the codebase, we've used:

    import static com.google.common.base.Preconditions.checkNotNull;

might want to use that for consistency.

Whatever you decide, you may want to resort the imports.

Reply:
+1 to that